app/assets/javascripts/application.js

```diff
@@ -112,6 +112,11 @@ function onContentLoad(){
 
   var tz = jstz.determine();
   $.cookie('timezone', tz.name(), { path: '/' });
+
+  // Removes all the templates from forms as they can't be cleared by strong parameters.
+  $('form').on('submit', function(e) {
```

Comment:
This only happens when editing Host interfaces, and Lookup keys right? Can it be moved there instead? Seems like a waste of time to do it everywhere.